lib/kitten_blue/jwk.ex

```diff
@@ -214,7 +214,7 @@ defmodule KittenBlue.JWK do
   def fetch!(jwks_uri) do
     case @http_client.get(jwks_uri) do
       {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
```

Comment:
BTW, I noticed that this line forces the HTTP client to HTTPoison. (I guess the HTTP client configuration is for Mox, though.)

Reply:
Thanks, I created issue for this problem. https://github.com/ritou/elixir-kitten-blue/issues/16